config/ts-dev.conf

```
@@ -58,8 +58,8 @@ location ~ \.php$ {
   fastcgi_param PHP_VALUE "memory_limit = 512M";
 
   fastcgi_intercept_errors on;
-  fastcgi_buffers 8 16k;
-  fastcgi_buffer_size 32k;
+  fastcgi_buffers 128 4096k;
+  fastcgi_buffer_size 4096k;
```

@olivierbouwman - did you do any testing on that page to see how big your buffer size needed to be in order for fastcgi to handle it? Bumping up these limits a little bit would be fine, but based on what I'm reading on these configs, the numbers here seem way way too big:

> By default, the buffer size is equal to one memory page. This is either 4K or 8K, depending on a platform. It can be made smaller, however.

Curious if you try something like this instead, how does it work for you?
```suggestion
  fastcgi_buffers 8 32k;
  fastcgi_buffer_size 64k;
```